### DISCUSSION TITLE
File upload limitation

### DISCUSSION THREAD

**Discussion by sy772** [docmost/docmost#290] [Help / Questions] [answered]
Hi are there any config file to change this file upload limit ? 
![image](https://github.com/user-attachments/assets/27eda817-7756-4d88-a220-088ac8720333)

**Comment by Philipinho** [comment]
Soon.

**Reply by sy772** [reply]
thank you for your quick response! best

**Reply by Philipinho** [reply, marked as answer]
You can now increase the file upload size limit in v0.4.0.
example usage:
```
FILE_UPLOAD_SIZE_LIMIT=100mb
```
